prow/cluster/starter.yaml

```diff
@@ -303,13 +303,17 @@ spec:
         - --config-path=/etc/config/config.yaml
         - --tide-url=http://tide/
         - --hook-url=http://hook:8888/plugin-help
+        - --github-token-path=/etc/github/oauth
```

Comment:
I suspect v20190828-ba003ad43 does not yet recognize the --github-token-path flag, correct? This should be in a follow up PR after we bump starter.yaml to a version that has this flag.

Reply:
It does, the flag got added some time ago by Miranda in https://github.com/kubernetes/test-infra/pull/13564